package-resources.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: secretgen-controller.carvel.dev.0.0.0
+spec:
+  refName: secretgen-controller.carvel.dev
+  releasedAt: null
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - git: {}
+      template:
+      - ytt:
+          paths:
+          - config/package-bundle/config
+          - config/release
+      - kbld:
+          paths:
+          - config/release
+  valuesSchema:
+    openAPIv3: null
+  version: 0.0.0
+
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: secretgen-controller.carvel.dev
+spec:
+  displayName: secretgen-controller
+  longDescription: secretgen-controller.carvel.dev
```

Comment:
These values are different from what they are releasing with today. It has to be same.

Reply:
Updated PackageMetada